docker/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM debian:9.9-slim
 # Default google-fluentd to the latest stable.
 # Use `docker build --build-arg REPO_SUFFIX=<CUSTOM_REPO_SUFFIX>` to override
 # the version.
-ARG REPO_SUFFIX=20190829-1
+ARG REPO_SUFFIX=20190915-1
```

Comment:
why the date is 0915 not today's date?

Reply:
That's the date we built the latest release that passed our integration tests.